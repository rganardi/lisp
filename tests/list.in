(define list (lambda x x))

(list (quote a) (quote b) list (quote 0))
