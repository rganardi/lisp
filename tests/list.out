parse_eval: unspecified
(a b (lambda x x) 0)
